src/utilities/utility.ts

```diff
@@ -74,16 +74,23 @@ export function annotateChildPods(kubectl: Kubectl, resourceType: string, resour
 }
 
 export function annotateNamespace(kubectl: Kubectl, namespaceName: string): IExecSyncResult {
-    let annotate = true;
     const result = kubectl.getResource('namespace', namespaceName);
-    this.checkForErrors([result]);
-    const annotationsSet = JSON.parse(result.stdout).metadata.annotations;
-    if (!!annotationsSet && !!annotationsSet.runUri && annotationsSet.runUri.indexOf(process.env['GITHUB_REPOSITORY']) == -1) {
-        annotate = false;
-        core.debug(`Skipping 'annotate namespace' as namespace annotated by other workflow`);
+    if (result == null) {
+        return <IExecSyncResult>{ code: -1, stderr: 'Failed to get resource' };
     }
-    if (annotate) {
-        return kubectl.annotate('namespace', namespaceName, workflowAnnotations, true);
+    else if (!!result && !!result.stderr) {
+        return result;
+    }
+
+    if (!!result && !!result.stdout) {
+        const annotationsSet = JSON.parse(result.stdout).metadata.annotations;

```

Comment:
Add null checks

Reply:
responded above for the same.

Reply:
I meant the null checks for `.metadata` and `.annotations`

Reply:
`metadata` is mandatory object and `annotation` check is added in next line.